src/ext/java/org/opentripplanner/ext/gtfsgraphqlapi/datafetchers/VehiclePositionImpl.java

```diff
@@ -21,17 +21,19 @@ public DataFetcher<String> label() {
 
   @Override
   public DataFetcher<Long> lastUpdated() {
-    return env -> getSource(env).time().getEpochSecond();
```

Comment:
Since this Impl is specifically about positions, I think we should make a separate method on the service where we fetch a different type (`RealtimeVehiclePosition`) and these are guaranteed to have a time, coordinates and a vehicleId.

Reply:
Timestamp is actually an optional field in vehicle positions. I don't think we should have a fallback value for it. I think this vehicle positions type might be useful also if someone opts to include STOP_POSITION but not POSITION from the new features selection.

Reply:
At the very least make a resuable method. However, I think the best solution would be to return `Optional`s. 

Still, I would filter out those that only have occupancy information.

What is the feature configuration that _you_ are planning to use? I don't want to introduce configurability and complexity for a feature that neither of us wants to use.

Reply:
I've used optionals now